spec.html

```diff
@@ -24392,20 +24392,27 @@ <h1>
       <emu-alg>
         1. Let _accessObj_ be OrdinaryObjectCreate(%Object.prototype%).
         1. If _kind_ is ~field~, ~method~, ~accessor~, or ~getter~, then
-          1. Let _getterClosure_ be a new Abstract Closure with no parameters that captures _name_ and performs the following steps when called:
-            1. Let _o_ be the *this* value.
-            1. If _name_ is a String, return ? Get(_o_, _name_).
-            1. Else, return ? PrivateGet(_name_, _o_).
-          1. Let _getter_ be CreateBuiltinFunction(_getterClosure_, 0, *""*, &laquo; &raquo;).
+          1. Let _getterClosure_ be a new Abstract Closure with parameter (_obj_) that captures _name_ and performs the following steps when called:
+            1. If _obj_ is not an Object, throw a *TypeError* exception.
+            1. If _name_ is either a String or a Symbol, return ? Get(_obj_, _name_).
```

Comment:
The change to accept Symbols looks good, but just to confirm my understanding, that's a bugfix unrelated to the normative changes presented in committee, right?

(I have no problem with it going in with this PR, I just want to make sure I understand why it's here.)

Reply:
Yes, its a bugfix. Without it, we'd try to do a `PrivateGet` using a symbol and not a Private Name, since _name_ is already declared to be a String, Symbol, or Private Name in the AO.

Reply:
> (I have no problem with it going in with this PR, I just want to make sure I understand why it's here.)

It's a bit of a drive-by bugfix since I needed to rewrite the algorithm steps for both `get` and `set` anyways and noticed the oversight.

Reply:
> Yes, its a bugfix. Without it, we'd try to do a `PrivateGet` using a symbol and not a Private Name, since _name_ is already declared to be a String, Symbol, or Private Name in the AO.

Correction: _name_ could already be a String, Symbol, or Private Name per the _name_ argument to the `CreateDecoratorContextObject` AO.